cppForSwig/Makefile

```diff
@@ -40,23 +42,33 @@ else
 PYTHON_INCLUDES=$(shell $(PYVER)-config --includes )
 endif
 
-CPPFLAGS += $(ARMORY_CPPFLAGS) -Icryptopp -Imdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
-LDLIBS += -lpthread -Lmdb
-SWIG_OPTS    += -c++ -python -threads
-
-SWIG_INC     += 
-
-CXXCPP += $(CPPFLAGS) $(PYTHON_INCLUDES) -std=c++11
+# "-Wl,--no-as-needed" used for Ubuntu 12.04 due to rt library linking issues.
+# http://stackoverflow.com/questions/17150075/undefined-reference-to-clock-gettime-although-lrt-is-given
+# has more info. Require only for static linking. When 12.04 goes, remove this
+# unless it's discovered that OS X needs it.
+ifdef STATIC_LINK
+LDLIBS += -Wl,--no-as-needed
```

Comment:
@droark - Sounds good.

When you talk about pushing patches both here and to 0.93-bugfix, do you mean the etotheipi 0.93-bugfix branch? Because I don't think I have access to push there. Or do you mean I should file another PR against etotheipi:0.93-bugfix instead of etotheipi:dev?